theatres/aliabadregion-2024/TCO30_Austin_JanitorialDuty.MCN_Aliabad/edit_me/briefing/mission.html

```diff
@@ -0,0 +1,15 @@
+Earlier today, a raid was conducted to capture and extract several HVTs from a remote compound south-east of Ahmad Golam.
+We have three names:
+
+-Saif Uallah
+-Abdul-Bari
+-Maryam Sayed
+
+However, resistance faced during our raid proved far greater than we could handle. Our HVTs were able to escape in the confusion.
+The CIA isn't keen on losing this opportunity, but they do recognize the increased risk of going back in.
+So, all they're hoping to snatch is Saif Uallah.
```

Comment:
```suggestion
However, resistance faced during our raid proved far greater than we could handle. Our HVTs were able to escape in the confusion. The CIA isn't keen on losing this opportunity, but they do recognize the increased risk of going back in.
So, all they're hoping to snatch is Saif Uallah.
```

The single sentence breaks might look better in a text editor, but look weird in Arma